py-polars/polars/lazyframe/frame.py

```diff
@@ -3648,6 +3648,60 @@ def rename(self, mapping: dict[str, str]) -> Self:
         new = list(mapping.values())
         return self._from_pyldf(self._ldf.rename(existing, new))
 
+    def insert_at_index(self, index: int, series: Series) -> Self:
```

Comment:
inserting a Series into a LazyFrame doesn't look right, does it? maybe I should leave this out